Doc/Tutorial/chapter_quick_start.tex

```diff
@@ -16,7 +16,7 @@ \section{General overview of what Biopython provides}
 \section{Working with sequences}
 \label{sec:sequences}
 
-Disputably (of course!), the central object in bioinformatics is the sequence. Thus, we'll start with a quick introduction to the Biopython mechanisms for dealing with sequences, the \verb|Seq| object, which we'll discuss in more detail in Chapter~\ref{chapter:Bio.Seq}.
+Disputably (Of course!), the central object in bioinformatics is the sequence. Thus, we'll start with a quick introduction to the Biopython mechanisms for dealing with sequences, the \verb|Seq| object, which we'll discuss in more detail in Chapter~\ref{chapter:Bio.Seq}.
```

Comment:
There is an exclamation mark which means it's a sentence. I'm not native speaker, but  generally speaking, sentences starts with a capital letter. I think.